src/worker.cc

```diff
@@ -480,12 +480,12 @@ void parse_contours(const rapidjson::Document& doc,
   }
 }
 
-void from_json(rapidjson::Document& doc, DirectionsOptions& options) {
+void from_json(rapidjson::Document& doc, Options& options) {
   bool track = !options.has_do_not_track() || !options.do_not_track();
 
   // TODO: stop doing this after a sufficient amount of time has passed
-  // move anything nested in deprecated directions_options up to the top level
-  auto deprecated = get_child_optional(doc, "/directions_options");
```

Comment:
this was not supposed to be changed and breaks backwards compatability for deprecated options request specification. fixing..